CHANGELOG.md

```diff
@@ -241,7 +241,7 @@ released.
 * [BREAKING] Raise.Action() methods removed. Use Raise.Event<THandler>() (e.g. Raise.Event<Action>>()).
 * [BREAKING] Renamed Raise.Event<TEventArgs>() methods to Raise.EventWith<TEventArgs>().
 * [UPDATE] Arg matchers can be specified using more specific, compatible type (Arg.Is<string>(..) for arg of type object).
-* [NEW] NSubstitute website and documentation http://nsubstitute.github.com
+* [NEW] NSubstitute website and documentation https://nsubstitute.github.com
```

Comment:
I wasn't sure about changing this due to the historical nature of the changelog. What do you think?
If we're going to change it should probably go to `https://nsubstitute.github.io`.

Reply:
The internet lives and so links get out-to-date, so nothing wrong to change them if that won't change the content

and yes, it should be .io 👼

Reply:
changed to io